datadog.cabal

```diff
@@ -17,14 +17,21 @@ build-type:          Simple
 cabal-version:       >=1.10
 
 library
-  exposed-modules:     Network.Datadog.StatsD
+  exposed-modules:     Network.Datadog,
+                       Network.Datadog.Check,
+                       Network.Datadog.Downtime,
+                       Network.Datadog.Event,
+                       Network.Datadog.Host,
+                       Network.Datadog.Monitor
+                       Network.Datadog.StatsD
                        -- Datadog.Metrics,
                        -- Datadog,
                        -- Datadog.Wai
-  -- other-modules:       
+  other-modules:       Network.Datadog.Internal
   other-extensions:    OverloadedStrings, GeneralizedNewtypeDeriving, TemplateHaskell, FunctionalDependencies, MultiParamTypeClasses, FlexibleInstances
   build-depends:       base >=4.7 && <5,
                        aeson >=0.8 && <0.10,
+                       http-conduit >=2.1 && <2.2,
```

Comment:
I'd prefer to avoid introducing a dependency on `http-conduit`. Could you replace the use of httpLbs in the internal module with `fmap fromChunks . brConsume . responseBody` so we can just use `http-client`?